lib/src/platform.ts

```diff
@@ -118,8 +118,12 @@ export class PlatformClient {
  */
 function createAuthInterceptor(authProvider: AuthProvider): Interceptor {
   const authInterceptor: Interceptor = (next) => async (req) => {
+    const url = new URL(req.url);
+    const pathOnly = url.pathname;
+    // TODO RCP: discuss in the PR if this is how we want to do it
+    // Signs only the path of the url in the request
     const token = await authProvider.withCreds({
-      url: req.url,
+      url: pathOnly,
```

Comment:
Signing only the path of the request fixes Dpop issue. With http gateway, the DPOP verification expects the whole url, ex: `https://host.com/v2/rewap`. But for RPC request it only expecting the path.

Reply:
I think this is something we need to fix in the auth middleware.

Reply:
I think so too. I've added a task in the backlog to at least add a good example for Auth Interceptor with Dpop